src/main/g8/client/src/main/scala/PeopleServiceClient.scala

```diff
@@ -0,0 +1,56 @@
+package $package$.client
+
+import cats.effect._
+import cats.syntax.applicative._
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import $package$.protocol.services._
+import io.grpc.{CallOptions, ManagedChannel}
+import monix.execution.Scheduler
+import io.chrisdavenport.log4cats.Logger
+import scala.concurrent.duration._
+
+trait PeopleServiceClient[F[_]] {
+
+  def getPerson(name: String): F[Person]
+
+}
+object PeopleServiceClient {
+
+  def apply[F[_]: Effect](clientF: F[PeopleService.Client[F]])(
+      implicit L: Logger[F]): PeopleServiceClient[F] =
+    new PeopleServiceClient[F] {
+
+      def getPerson(name: String): F[Person] =
+        for {
+          client <- clientF
+          _ <- L.info(s"Request: \$name")
```

Comment:
Do you want to escape the `$` character?

Reply:
Yes. This is the code that Giter8 needs to compile, the way how g8 usually detects variables is using `$`, so all the string-interpolation inside of the `g8` needed to be scaped :-(

Reply:
I see, got it 👍